net/ampr-ripd/Makefile

```diff
@@ -0,0 +1,97 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=ampr-ripd
+PKG_VERSION:=2.4
+PKG_RELEASE:=1
+PKG_SOURCE_URL:=https://www.yo2loj.ro/hamprojects
+PKG_HASH:=18d00c898d22a5ee1fb2c153a49358100b5ff22b2b57668472a651998fb0bf6f
+PKG_MAINTAINER:=Dan Srebnick <k2ie@k2ie.net>
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tgz
+PKG_LICENSE:=GPL-2.0-only
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/ampr-ripd
+	SECTION:=net
+	CATEGORY:=Network
+	SUBMENU:=Routing and Redirection
+	DEPENDS:=+kmod-ipip +ip-full
+	TITLE:=Routing daemon for the AMPR network
+	URL:=https://www.yo2loj.ro/hamprojects
+endef
+
+define Package/ampr-ripd/description
+	Routing daemon written in C similar to Hessu's rip44d including optional resending of RIPv2 broadcasts for router injection.
+endef
+
+CONFIGURE_VARS+= \
+	CC="$(TOOLCHAIN_DIR)/bin/$(TARGET_CC)"
+	COPT="$(TARGET_COPT)"
+
+define Package/ampr-ripd/install
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/ampr-ripd $(1)/usr/sbin
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/ampr-ripd-init $(1)/etc/init.d/ampr-ripd
+	$(INSTALL_DIR) $(1)/etc/uci-defaults
+	$(INSTALL_DATA) ./files/99-ampr-ripd $(1)/etc/uci-defaults/99-ampr-ripd
+endef
+
+define Package/ampr-ripd/postrm
```

Comment:
With opkg `postrm` is called _before_ an upgrade with `PKG_UPGRADE=1`. So if you don't want all of the configs to be deleted on each upgrade, you should take this into account, e.g.:

```bash
[ "$${PKG_UPGRADE}" = 1 ] && exit 0
```

Reply:
Those should be marked as conffiles. :)